common/auth_backend.py

```diff
@@ -0,0 +1,24 @@
+from authbroker_client.backends import AuthbrokerBackend
+from django.contrib.auth import get_user_model
+
+UserModel = get_user_model()
+
+
+class CustomAuthbrokerBackend(AuthbrokerBackend):
+    def user_create_mapping(self, profile):
+        return {
+            "is_active": True,
+            "first_name": profile["first_name"],
+            "last_name": profile["last_name"],
+            "sso_uuid": profile["user_id"],
+        }
+
+    def confirm_sso_uuid(self, user, profile):
+        if not user.sso_uuid:
+            user.sso_uuid = profile["user_id"]
```

Comment:
It would be safer to either wrap this in a try / except (perhaps with a `logger.warning()` in the except block) or do:
```suggestion
            user.sso_uuid = profile.get("user_id")
```
I know the base class applies your approach, but we'd want to reasonably avoid application failure due to base class changes, and we can do that here at almost no cost.